contrib/clojure-package/README.md

```diff
@@ -1,110 +1,125 @@
+To maximize leverage, the Clojure package has been built on the existing Scala package using interop. This approach has allowed rapid initial development and close parity with the Scala package functionality. It also leaves the door open to directly developing code against the JNI bindings with Clojure in the future in an incremental fashion, using the test suites as a refactoring guide.
 
 For a **video introduction**, see [Clojure MXNet with Carin Meier - Clojure Virtual Meetup](https://www.crowdcast.io/e/clojure-mxnet-with-carin) (setup instructions from 20:49)
 
 ## Current State and Plans
 
-Help is needed testing and generally making the package better. A list of the pacakge status and contribution needs can be found here [Clojure Package Contribution Needs](https://cwiki.apache.org/confluence/display/MXNET/Clojure+Package+Contribution+Needs). Please get involved :)
-
+Help is needed testing and generally making the package better. A list of the pacakge status and contribution needs can be found under [Clojure Package Contribution Needs](https://cwiki.apache.org/confluence/display/MXNET/Clojure+Package+Contribution+Needs). Please get involved :)
 
 ## Getting Started
 
-The Clojure MXNet framework consists of a core C library, a Scala Api that talks to it through JNI bindings, and finally a Clojure wrapper around the Scala api.
+The Clojure MXNet framework consists of a core C library, a Scala API that talks to the core through [JNI (Java Native Interface)](https://en.wikipedia.org/wiki/Java_Native_Interface) bindings, and finally a Clojure wrapper around the Scala API.
 
 Since there is a native code involved in the framework, what OS you are running matters.
 
-The following systems are supported:
+The following combinations of operating system and compute device are supported:
 
-- OSX cpu
-- Linux cpu
-- Linux gpu
+- Linux CPU
+- Linux GPU
+- OSX CPU
 
 There are three ways of getting started:
 
-* Use the Clojure jars with the native dependencies baked in. This the easiest way to get going.
-* Checkout the MXNet project from a release tag using the Scala jars with native deps. This is also a pretty easy way to get started.
-* Build from the MXNet project master. This option can be used to build the whole project yourself.
+1. Install [prebuilt Clojure jars](https://search.maven.org/search?q=clojure%20mxnet) with the native dependencies baked in. This the quickest way to get going.
+2. Install the Clojure package from source, but use prebuilt jars for the native dependencies. Choose this option if you want pre-release features of the Clojure package but don't want to build (compile native dependencies yourself.
```

Comment:
```suggestion
2. Install the Clojure package from source, but use prebuilt jars for the native dependencies. Choose this option if you want pre-release features of the Clojure package but don't want to build (compile) native dependencies yourself.
```

Reply:
That's great. I think there will be some merge conflicts with the current version that puts anchors in.